### PR TITLE
Add templates for bug reports and feature requests

### PR DESCRIPTION
**Pull request by ColCarroll**
More tidying of the issue templates. I actually merged this and enabled issues on my fork, so you can [play around with them there](https://github.com/ColCarroll/arviz/issues) if you'd like, but the `Issues` screen now looks like this:

image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+name: 'Feature Request'
+about: Suggest an idea for this project
+
+---
+
+## Tell us about it
+
+The more specific the better. You can assume you have access to any combination of datasets defined in the schema. When considering an implementation, keep in mind that
+
+- models in ArviZ can have one, or lots of random variables,
```
Comment (canyon289) [inline .github/ISSUE_TEMPLATE/feature-request.md]: Being picky here. If we're doing a comma I think we need another comma, but only partially sure of this
```suggestion
- models in ArviZ can have one, or lots, of random variables,
```

**Comment by canyon289** [comment]
This is way better. LGTM

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 1 above)
Comment (ColCarroll) [reply .github/ISSUE_TEMPLATE/feature-request.md]: good catch.

**Comment by canyon289** [comment]
Merging

**Pull request merged**